src/useImage.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useState} from 'react'
+import React, {useState, useEffect} from 'react'
```

Comment:
Looks like this `useEffect` is never used.

Reply:
fixed